// emscripten-wasm-loader/dist/cjs/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var environment_1 = require("./environment");
exports.ENVIRONMENT = environment_1.ENVIRONMENT;
var getModuleLoader_1 = require("./getModuleLoader");
exports.getModuleLoader = getModuleLoader_1.getModuleLoader;
var logger_1 = require("./util/logger");
exports.enableLogger = logger_1.enableLogger;
exports.log = logger_1.log;
var isNode_1 = require("./util/isNode");
exports.isNode = isNode_1.isNode;
var isWasmEnabled_1 = require("./util/isWasmEnabled");
exports.isWasmEnabled = isWasmEnabled_1.isWasmEnabled;
var isMounted_1 = require("./path/isMounted");
exports.isMounted = isMounted_1.isMounted;
var mkdirTree_1 = require("./path/mkdirTree");
exports.mkdirTree = mkdirTree_1.mkdirTree;
var mountBuffer_1 = require("./path/mountBuffer");
exports.mountBuffer = mountBuffer_1.mountBuffer;
var mountDirectory_1 = require("./path/mountDirectory");
exports.mountDirectory = mountDirectory_1.mountDirectory;
var unmount_1 = require("./path/unmount");
exports.unmount = unmount_1.unmount;
